compiler/ballerina-parser/src/test/resources/statements/var-decl-stmt/local_var_decl_source_17.bal

```diff
@@ -0,0 +1,4 @@
+function myFunc() {
+    int a = 1;
+    a
```

Comment:
We need to fix this for qualified name ref as well. Say we have defined a variable in another module. Then we still get this "unknown type" error when written. e.g. `module1:a`

Reply:
Updated 847cb19a2db34fe680d2f3678a748557ee8c8431